docs/docs-content/clusters/pcg/manage-pcg/migrate_pcg.md

```diff
@@ -0,0 +1,155 @@
+---
+sidebar_label: "Migrate a PCG"
+title: "Migrate a PCG"
+description: "Learn why, when, and how to migrate an existing Private Cloud Gateway (PCG) to a differnt PCG."
+hide_table_of_contents: false
+sidebar_position: 50
+tags: ["pcg"]
+---
+
+Clusters deployed through a Private Cloud Gateway (PCG) maintain a reference point to the PCG that deployed them. This
+reference point is needed so that Palette knows what PCG to use when the cluster needs to be removed. A Cluster API
```

Comment:
```suggestion
reference point is needed so that Palette knows what PCG to use when the cluster needs to be removed. 
A Cluster API
```
I would start a new paragraph here.